packages/core/services/AnnotationService/HttpAnnotationService/index.ts

```diff
@@ -30,7 +31,10 @@ export default class HttpAnnotationService extends HttpServiceBase implements An
         console.log(`Requesting annotation values from ${requestUrl}`);
 
         const response = await this.get<AnnotationResponse>(requestUrl);
-        return map(response.data, (annotationResponse) => new Annotation(annotationResponse));
+        return [
```

Comment:
Previously we had referencing to `TOP_LEVEL_FILE_ANNOTATIONS` scattered around components because they needed to optionally do something special to render `TOP_LEVEL_FILE_ANNOTATIONS` since they weren't treated as real annotations. By returning `TOP_LEVEL_FILE_ANNOTATIONS` in the HTTP specific services we can avoid having the components or selectors worry about whether or not the app is currently querying HTTP or DuckDB.